kernel/tests/read.rs

```diff
@@ -987,18 +988,60 @@ fn with_predicate_and_removes() -> Result<(), Box<dyn std::error::Error>> {
 #[test]
 fn short_dv() -> Result<(), Box<dyn std::error::Error>> {
     let expected = vec![
-        "+----+-------+-------------------------+---------------------+",
-        "| id | value | timestamp               | rand                |",
-        "+----+-------+-------------------------+---------------------+",
-        "| 3  | 3     | 2023-05-31T18:58:33.633 | 0.7918174793484931  |",
-        "| 4  | 4     | 2023-05-31T18:58:33.633 | 0.9281049271981882  |",
-        "| 5  | 5     | 2023-05-31T18:58:33.633 | 0.27796520310701633 |",
-        "| 6  | 6     | 2023-05-31T18:58:33.633 | 0.15263801464228832 |",
-        "| 7  | 7     | 2023-05-31T18:58:33.633 | 0.1981143710215575  |",
-        "| 8  | 8     | 2023-05-31T18:58:33.633 | 0.3069439236599195  |",
-        "| 9  | 9     | 2023-05-31T18:58:33.633 | 0.5175919190815845  |",
-        "+----+-------+-------------------------+---------------------+",
+        "+----+-------+--------------------------+---------------------+",
+        "| id | value | timestamp                | rand                |",
+        "+----+-------+--------------------------+---------------------+",
+        "| 3  | 3     | 2023-05-31T18:58:33.633Z | 0.7918174793484931  |",
+        "| 4  | 4     | 2023-05-31T18:58:33.633Z | 0.9281049271981882  |",
+        "| 5  | 5     | 2023-05-31T18:58:33.633Z | 0.27796520310701633 |",
+        "| 6  | 6     | 2023-05-31T18:58:33.633Z | 0.15263801464228832 |",
+        "| 7  | 7     | 2023-05-31T18:58:33.633Z | 0.1981143710215575  |",
+        "| 8  | 8     | 2023-05-31T18:58:33.633Z | 0.3069439236599195  |",
+        "| 9  | 9     | 2023-05-31T18:58:33.633Z | 0.5175919190815845  |",
+        "+----+-------+--------------------------+---------------------+",
     ];
     read_table_data_str("./tests/data/with-short-dv/", None, None, expected)?;
     Ok(())
 }
+
```

Comment:
these are taken from the delta-java tests. i validated that the `expected` arrays here are correct.